extensions/ApiExplorer/SpecialApiExplorer.php

```diff
@@ -50,7 +50,7 @@ public function execute( $par = null ) {
 		wfProfileIn( __METHOD__ );
 
 		// TODO: Make this work for ResourceLoader (Wikia isn't using RL yet at the time of this writing).
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app